src/doc/src/reference/environment-variables.md

```diff
@@ -35,11 +35,17 @@ system:
 * `RUSTDOCFLAGS` — A space-separated list of custom flags to pass to all `rustdoc`
   invocations that Cargo performs. In contrast with [`cargo rustdoc`], this is
   useful for passing a flag to *all* `rustdoc` instances. See
-  [`build.rustdocflags`] for some more ways to set flags.
+  [`build.rustdocflags`] for some more ways to set flags. This string is
+  split by whitespace; for a more robust encoding of multiple arguments,
+  set `CARGO_ENCODED_RUSTDOCFLAGS` instead with arguments separated by
+  `0x1f` (ASCII Unit Separator).
```

Comment:
Maybe use `0xff` instead? That is not valid UTF-8 and as such is not allowed in arguments to rustc. `0x1f` is and as such could be used eg in `-Clink-arg` or `--extern`.

Reply:
Do all platforms support arbitrary bytes in environment variables? Using invalid UTF-8 would also mean you'd have to consume it with `env::var_os` as an `OsString`, which is more annoying to process without methods like `split`.

Reply:
Yeah, I also consider the delimiter being valid UTF-8 a benefit. It's true that someone _could_ used `0x1f` in an argument, but I don't see how anyone would in practice?